R/run-url.R

```diff
@@ -14,6 +14,7 @@
 #' @param subdir A subdirectory in the repository that contains the app. By
 #'   default, this function will run an app from the top level of the repo, but
 #'   you can use a path such as `\code{"inst/shinyapp"}.
+#' @param destDir Directory to store the downloaded application files
```

Comment:
It would be helpful to add, 

"If \code{NULL} (the default), the application files will be stored in a temporary directory and removed when the app exits."